cli/module_build.go

```diff
@@ -1,16 +1,17 @@
 package cli
 
 import (
-	"context"
 	"fmt"
 	"io"
 	"text/tabwriter"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/urfave/cli/v2"
+	"go.uber.org/multierr"
 	buildpb "go.viam.com/api/app/build/v1"
 	"go.viam.com/utils/pexec"
+	"golang.org/x/exp/slices"
```

Comment:
slices lands for real in 1.21

Reply:
though we're going to be on 1.21 soon, okay IMO to use this one for now and update when possible

Reply:
We use `x/exp` in 11 places in rdk (`grep -r "x/exp" | grep "\.go" | wc -l`).

I think we assume it is semi-reliable.


When is the switch to 1.21 coming? Today/this week?